src/generator.py

```diff
@@ -137,7 +145,19 @@ def generate_template_at_destination(self, destination: pathlib.Path) -> None:
         ProjectTemplateAndDestinationChecker(self.template, destination).check()
         copy_directory_and_contents_to_new_location(self.template.template_directory, destination)
         if self.template.shared_files_directory is not None:
-            copy_directory_and_contents_to_new_location(self.template.shared_files_directory, destination)
+            copy_directory_and_contents_to_new_location(self.template.shared_files_directory,
+                                                        destination,
+                                                        just_files=True)
+            if self.template.cicd_type == 'github':
+                cicd_target = destination / '.github' / 'workflows'
```

Comment:
@asbfall  Regarding the .github / workflow files, we are copying the file to this folder if the 
`--cicd 'github'` flag is applied (Default)